wd_core.au3

```diff
@@ -314,17 +306,19 @@ EndFunc   ;==>_WD_Status
 ;                  Failure - "" (empty string) and sets @error to $_WD_ERROR_Exception
 ; Author ........: Danp2
 ; Modified ......:
-; Remarks .......: The Get Session functionality was added and then removed from the W3C draft spec, so the code is commented
-;                  until they determine how this should function. See w3c/webdriver@35df53a for details. Meanwhile, I temporarily
-;                  changed the code to return the information that is available
-; Related .......: _WD_CreateSession
+; Remarks .......: The Get Session functionality was added and then removed from the W3C draft spec,
+;                  so the code is commented until they determine how this should function.
+;                  See w3c/webdriver@35df53a for details.
+;                  Meanwhile, I temporarily changed the code to return the information that is available.
+; Related .......: _WD_CreateSession, _WD_LastHTTPResult
 ; Link ..........: https://www.w3.org/TR/webdriver#get-session
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_GetSession($sSession)
 	Local Const $sFuncName = "_WD_GetSession"
 	Local $sResult
 	#forceref $sSession, $sFuncName
+	$_WD_HTTPRESULT = 0

```

Comment:
- I'm wondering why the remarks were changed. Was there a problem with the formatting?
- I don't think we need to reset $_WD_HTTPRESULT here

Reply:
> * I'm wondering why the remarks were changed. Was there a problem with the formatting?

It is easier to read.

Reply:
> * I don't think we need to reset $_WD_HTTPRESULT here

Agree. Look on: [_WD_GetSession](https://github.com/Danp2/au3WebDriver/pull/247/commits/2cb12f155fcf4af4b2a22d85476f9d782275860b)

Reply:
> It is easier to read.

Not a big deal, but I disagree.

Reply:
Reverted. Take a look on: [_WD_GetSession remarks reverted](https://github.com/Danp2/au3WebDriver/pull/247/commits/aaf07e308d27921cd1fbd23b705acb779c377f85)